Firmware/src/network/WifiManager.h

```diff
@@ -38,16 +38,21 @@ class WifiManager {
 
   void connectToWiFi() {
     int timeOut_s = 30;
+    WiFiClass::hostname(this->networkConfig->getHostname().c_str());
 
-    if (config->getSSID().isEmpty()) {
+    if (networkConfig->getSsid().isEmpty() || !config->isInitailised()) {
+      LV_LOG_INFO("Config is not initailised starting AP mode");
       ap = new WifiAccessPoint();
       return;
     }
     LV_LOG_INFO("We have config - let's try STA");
 
-    delete ap;
-    ap = nullptr;
-    sta = new WifiStation(config);
+    if (ap != nullptr) {
+      delete ap;
```

Comment:
delete on nullptr should do nothing - so check is an excessive AFAIK

Reply:
Tested and push'd